src/EditorFeatures/Test2/Expansion/NameExpansionTests.vb

```diff
@@ -222,7 +222,7 @@ class C
             Await TestAsync(input, expected, expandParameter:=True)
         End Function
 
-        <Fact, Trait(Traits.Feature, Traits.Features.Expansion)>
+        <Fact(Skip:="https://github.com/dotnet/roslyn/issues/11979"), Trait(Traits.Feature, Traits.Features.Expansion)>
```

Comment:
Please submit the change to this file as a separate PR because it is not related to the actual goal of this change.